mobile/src/screens/ChatScreen.tsx

```diff
@@ -1,21 +1,31 @@
 import React from 'react';
 import { Text, View, StyleSheet } from 'react-native';
+import { TextStyles } from '../styles/TextStyles';
 
 /**
  * Left tab on navbar for chatting with volunteers
  */
 export const ChatScreen: React.FC = () => {
   return (
     <View style={styles.container}>
-      <Text>Chat</Text>
+      <Text style={TextStyles.h3}>  {`\tLet's Talk\n`}</Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Tel: </Text><Text>858.274.9673{`\n`}</Text> </Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Fax: </Text><Text>858.274.9673{`\n`}</Text> </Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Mon-Fri: </Text><Text>8am-4pm{`\n`}</Text> </Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Email Us: </Text><Text>info@wordsalive.gov{`\n`}</Text> </Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Address: </Text><Text>{`5111 Santa Fe Street Suite\n\t219 San Diego, California, 92081, \n\tUnited States`}{`\n`}</Text> </Text>
     </View>
+    
   );
 };
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    alignItems: 'center',
+    alignItems: 'flex-start',
     justifyContent: 'center',
+    backgroundColor: 'orange',
+    width: '400pt',
+    height: '400pt',
```

Comment:
This is crashing my phone, I don't think "pt" is a unit in RN. What is this supposed to do?